scripts/prepare-code-coverage-artifact.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Concord
+#
+# Copyright (c) 2021 VMware, Inc. All Rights Reserved.
+#
+# This product is licensed to you under the Apache 2.0 license (the "License").
+# You may not use this product except in compliance with the Apache 2.0 License.
+#
+# This product may include a number of subcomponents with separate copyright
+# notices and license terms. Your use of these subcomponents is subject to the
+# terms and conditions of the subcomponent's license, as noted in the LICENSE
+# file.
+
+from __future__ import print_function
+
+import argparse
+import os
+import subprocess
+import sys
+import shutil
+
+def get_llvm_tools(coveragedir):
+    """
+    Return the path of llvm-profdata and llvm-cov
+    """
+    profdata_exe = os.path.join(coveragedir, 'llvm-profdata')
+    cov_exe = os.path.join(coveragedir, 'llvm-cov')
+    if os.path.exists(profdata_exe):
+        return profdata_exe, cov_exe
+    if os.path.exists('/usr/lib/llvm-7/bin/llvm-profdata'):
```

Comment:
in order to prevent ambiguity, I would suggest choosing the latest LLVM version available in the current docker image.

Reply:
Changes have been added to take default llvm-profdata and llvm-cov binaries from docker images.